src/main/java/me/ryanhamshire/GriefPrevention/events/ClaimInspectionEvent.java

```diff
@@ -0,0 +1,62 @@
+package me.ryanhamshire.GriefPrevention.events;
+
+import me.ryanhamshire.GriefPrevention.Claim;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public class ClaimInspectionEvent extends PlayerEvent implements Cancellable
+{
+    private static final HandlerList handlers = new HandlerList();
+
+    public static HandlerList getHandlerList()
+    {
+        return handlers;
+    }
+
+    private final Collection<Claim> claims;
+    private final boolean shiftInspecting;
```

Comment:
I kinda dislike naming it after what key they'd be pressing (which would well have been remapped client-side) rather than what the effect is - e.g. `inspectingNearbyClaims` or something like that would be better, I think.